packages/common/src/pipes/number_pipe.ts

```diff
@@ -246,8 +266,8 @@ export class CurrencyPipe implements PipeTransform {
   }
 }
 
-function isEmpty(value: any): boolean {
-  return value == null || value === '' || value !== value;
+function isValue(value: number|string|null|undefined): value is number|string {
+  return !(value == null || value === '' || value !== value);
```

Comment:
Do we not prefer use of `isNaN`?

```suggestion
  return !(value == null || value === '' || isNaN(value));
```

Reply:
TypeScript rejects it because `value` is typed as `number|string` at that point.